third_party/cpgfunctionEP-0.3.2/third_party/eigen-3.3.9/Eigen/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+include(RegexUtils)
```

Comment:
Everything below here is eigen related. The library was stripped. The total number of files related to eigen went from 302 to 136.

Reply:
Seems like we'll now be carrying three copies of eigen in our third_party directory. Could we reuse eigen instead? or am I missing something?

Reply:
@Myoldmopar @mitchute It looks like egien-3.3.7 is locaed in `third_party/eigen`. Would it be okay for me to link to that copy of eigen?

Reply:
Yes sir, I'll see what I can do. Thank you.